### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#357]
You cannot access the swagger docs link in the navigation bar menu via keyboard. You get the divider instead. It should not hit on the divider and hit on the link

~~In command view, you also cannot navigate to remove items from a list ex echo_list_model list of strings~~

In request view, can't access the tooltip on the STATUS with only keyboard

 Can't access `x` button in global filter (table on Choose System/Choose Command for create job)

~~No keyboard interaction with the JSON cards at all, except open/close/download buttons in the card header~~

**Comment by west270** [comment]
Moved some of the items to separate ticket to be worked.
